conda/environments/all_cuda-120_arch-x86_64.yaml

```diff
@@ -1,6 +1,7 @@
 # This file is generated by `rapids-dependency-file-generator`.
 # To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
 channels:
+- numba/label/ci
```

Comment:
The channel only seems to show up environment files, which IIUC are only used for offline testing and development

For CI testing think we need this channel in CI files that `dependencies.yaml` doesn't affect AFAIK